.github/workflows/ci.yaml

```diff
@@ -11,6 +11,10 @@ on:
   schedule:
     - cron:  '0 17 * * *'
 
+concurrency:
+  group: '${{ github.head_ref || github.ref }}-ci'
```

Comment:
I don't understand why we are using `${{ github.head_ref || github.ref }}-ci` as the group name?
Can you please add a comment here?

Reply:
Added a comment above this line, lmk if that doesn't make sense.